espflash/src/cli/mod.rs

```diff
@@ -32,94 +29,99 @@ pub mod monitor;
+    /// Path to a CSV file containing partition table
     #[clap(long)]
-    pub erase_otadata: bool,
-}
-
-#[derive(Debug, Clone, Parser)]
-pub struct FlashConfigOpts {
-    /// Flash mode to use
-    #[clap(short = 'm', long, possible_values = FlashMode::VARIANTS, value_name = "MODE")]
-    pub flash_mode: Option<FlashMode>,
-    /// Flash size of the target
-    #[clap(short = 's', long, possible_values = FlashSize::VARIANTS, value_name = "SIZE")]
-    pub flash_size: Option<FlashSize>,
-    /// Flash frequency
-    #[clap(short = 'f', long, possible_values = FlashFrequency::VARIANTS, value_name = "FREQUENCY")]
-    pub flash_freq: Option<FlashFrequency>,
+    pub partition_table: Option<PathBuf>,
+    /// Load the application to RAM instead of Flash
+    #[clap(long)]
+    pub ram: bool,
 }
 
-#[derive(Clone, Debug, Parser)]
-pub struct PartitionTableOpts {
+/// Operations for partitions tables
+#[derive(Debug, Args)]
+pub struct PartitionTableArgs {
+    /// Optional output file name, if unset will output to stdout
+    #[clap(short = 'o', long)]
+    output: Option<PathBuf>,
+    /// Input partition table
+    partition_table: PathBuf,
     /// Convert CSV parition table to binary representation
-    #[clap(long, required_unless_present_any = ["info", "to-csv"])]
+    #[clap(long, conflicts_with = "to-csv")]
     to_binary: bool,
     /// Convert binary partition table to CSV representation
-    #[clap(long, required_unless_present_any = ["info", "to-binary"])]
+    #[clap(long, conflicts_with = "to-binary")]
     to_csv: bool,
-    /// Show information on partition table
-    #[clap(short, long, required_unless_present_any = ["to-binary", "to-csv"])]
-    info: bool,
-    /// Input partition table
-    partition_table: PathBuf,
-    /// Optional output file name, if unset will output to stdout
-    #[clap(short, long)]
-    output: Option<PathBuf>,
-}
-
-#[derive(Clone, Debug, Parser)]
-pub struct WriteBinToFlashOpts {
-    /// Address at which to write the binary file
-    #[clap(value_parser = parse_u32)]
-    addr: u32,
-
-    /// File containing the binary data to write
-    bin_file: String,
-
-    #[clap(flatten)]
-    connect_opts: ConnectOpts,
 }
 
-fn parse_u32(input: &str) -> Result<u32, ParseIntError> {
-    parse_int::parse(input)
+/// Save the image to disk instead of flashing to device
+#[derive(Debug, Args)]
+pub struct SaveImageArgs {
+    /// Custom bootloader for merging
+    #[clap(long)]
+    pub bootloader: Option<PathBuf>,
+    /// Chip to create an image for
+    #[clap(long, possible_values = Chip::VARIANTS)]
+    pub chip: Chip,
+    /// File name to save the generated image to
+    pub file: PathBuf,
+    /// Boolean flag to merge binaries into single binary
+    #[clap(long)]
+    pub merge: bool,
+    /// Custom partition table for merging
+    #[clap(long)]
+    pub partition_table: Option<PathBuf>,
+    /// Don't pad the image to the flash size
+    #[clap(long)]
+    pub skip_padding: bool,
```

Comment:
Please change this as per https://github.com/esp-rs/espflash/pull/241/

Reply:
These have been updated in [0c63abf](https://github.com/esp-rs/espflash/commit/0c63abf179b7ce158ce2f1445d1501a11ddf0905).